elements/lisk-p2p/src/p2p.ts

```diff
@@ -184,6 +184,15 @@ const createPeerPoolConfig = (
 	peerLists,
 });
 
+const filterAdvertiseAddressPeer = (peer: P2PPeerInfo) => {
```

Comment:
Why don't you make this a private P2P method? I would even aggregate the `filter` action inside the method. Otherwise it feels redundant `.filter(filterAdvertiseAddressPeer)`.